polymorphic/tests/test_orm.py

```diff
@@ -228,14 +228,14 @@ def test_defer_fields(self):
         objects_deferred = Model2A.objects.defer('field1')
 
         self.assertNotIn('field1', objects_deferred[0].__dict__, 'field1 was not deferred (using defer())')
-        self.assertEqual(repr(objects_deferred[0]),
-                         '<Model2A: id 1, field1 (CharField), deferred[field1]>')
-        self.assertEqual(repr(objects_deferred[1]),
-                         '<Model2B: id 2, field1 (CharField), field2 (CharField), deferred[field1]>')
-        self.assertEqual(repr(objects_deferred[2]),
-                         '<Model2C: id 3, field1 (CharField), field2 (CharField), field3 (CharField), deferred[field1]>')
-        self.assertEqual(repr(objects_deferred[3]),
-                         '<Model2D: id 4, field1 (CharField), field2 (CharField), field3 (CharField), field4 (CharField), deferred[field1]>')
+        self.assertRegex(repr(objects_deferred[0]),
```

Comment:
Honestly, this could really be an `assertQuerySetEqual`, which is more future-proof, and quite cleaner.

Not a blocker for this specific PR though.

Reply:
I'm not really sure how I would make that change. Thanks for not blocking on it!